project/profiles/capacity4more/behat/features/homepage.feature

```diff
@@ -126,9 +126,16 @@ Feature: Test homepage content and blocks
     Given I am logged in as user "isaacnewton"
     When  I visit the site homepage
     And   I select the radio button "My groups" with the id "edit-homepage-filter-groups"
-    Then  I should wait not to see "Nobel Prize" in the "div.activity-stream" element
```

Comment:
1. The test is checking for article activities visibility, not for the group activities.
2. The article activities text is different then what was written in the test:

![screen shot 2016-08-18 at 1 55 27 pm](https://cloud.githubusercontent.com/assets/13344923/17771412/70e4f99a-654b-11e6-8bcc-4f0e0c71755e.png)

![image](https://cloud.githubusercontent.com/assets/13344923/17771418/7a5cc0de-654b-11e6-9ef6-9fd6027b67e1.png)